lgsf/scrapers/base.py

```diff
@@ -1,13 +1,16 @@
 import abc
 import os
+
+import boto3
 from dateutil import parser
-import datetime
+from datetime import datetime
 import traceback
 
 import requests
-import requests_cache
 
-requests_cache.install_cache("scraper_cache", expire_after=60 * 60 * 24)
+# import requests_cache
```

Comment:
Commented out because it should probably be reinstated behind  a check for lambda env